lib/matplotlib/backend_bases.py

```diff
@@ -3218,14 +3218,71 @@ def press_pan(self, event):
         if (event.button not in [MouseButton.LEFT, MouseButton.RIGHT]
                 or event.x is None or event.y is None):
             return
-        axes = [a for a in self.canvas.figure.get_axes()
-                if a.in_axes(event) and a.get_navigate() and a.can_pan()]
-        if not axes:
+
+        # Get all axes and group them by their navigation-capture behaviour.
+        axes_capture = {True: {}, False: {}}
+        for a in self.canvas.figure.get_axes():
+            if a.in_axes(event) and a.get_navigate() and a.can_pan():
+                # By default, axes with an invisible patch forward events
```

Comment:
```suggestion
                # By default, events pass through invisible patches
```
I'm a bit confused by the wording choice of "forward". That word leaves it a bit ambiguous to me whether this axes handles the event and also passes it on, or if it just passes it on. I _think_ you're implying the first case where events simply pass through? I think this could also be changed in docstrings if you agree.